pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -103,6 +103,49 @@ func MonEndpoints(mons map[string]*cephclient.MonInfo, requireMsgr2 bool) []stri
 	return endpoints
 }
 
+// updateNetNamespaceFilePath modify the netNamespaceFilePath for all cluster IDs.
+// If holderEnabled is set to true. Otherwise, removes the netNamespaceFilePath value
+// for all the clusterIDs.
+func updateNetNamespaceFilePath(cc csiClusterConfig) {
+	var (
+		cephFSNetNamespaceFilePath string
+		rbdNetNamespaceFilePath    string
+		nfsNetNamespaceFilePath    string
+	)
+
+	if IsHolderEnabled() {
+		for _, centry := range cc {
+			if centry.CephFS.NetNamespaceFilePath != "" {
+				cephFSNetNamespaceFilePath = centry.CephFS.NetNamespaceFilePath
+			}
+			if centry.RBD.NetNamespaceFilePath != "" {
+				rbdNetNamespaceFilePath = centry.RBD.NetNamespaceFilePath
+			}
+			if centry.NFS.NetNamespaceFilePath != "" {
+				nfsNetNamespaceFilePath = centry.NFS.NetNamespaceFilePath
+			}
+		}
+
+		for i, centry := range cc {
+			if centry.CephFS.SubvolumeGroup != "" && centry.CephFS.NetNamespaceFilePath == "" {
+				cc[i].CephFS.NetNamespaceFilePath = cephFSNetNamespaceFilePath
+			}
+			if centry.RBD.RadosNamespace != "" && centry.RBD.NetNamespaceFilePath == "" {
+				cc[i].RBD.NetNamespaceFilePath = rbdNetNamespaceFilePath
```

Comment:
why subvolumegroup and radosnamespace check is required here?

Reply:
thought of adding netNamespaceFilePath only to clusterID which is created when subvolumegroup/radosnamespace is created.
Anyway, no harm in adding it I guess?

now removed the check.